apps/ejabberd/src/mod_mam_muc_cassandra_arch.erl

```diff
@@ -57,7 +57,7 @@
           room_jid :: binary(),
           nick_name :: binary(),
           with_nick :: binary(),
-          message :: binary()
+          message :: binary() | undefined
```

Comment:
@rslota, do we still need to add `undefined` since you changed the way the record is created in line 175?